chevah/geck
# [#1453] Add note about distributed tasks.

# Problem description

This review is about leaving something behind after the discussions about working from home that were held during our last meeting.

We should but in the public styleguide all our best practices and lessons learned while working from home (or not) in a distributed team.
# Changes description

There was already a page in styleguide with info about working from home.

I have extend it to talks about handing tasks in an distributed way.
# How to try and test the changes

reviewers @alibotean @lgheorghiu @dumol @bnickname 

Check that changes make sense.

Feel free to add your own notes.
